ios/pushprovisioning/PushProvisioningUtils.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PushProvisioningUtils.swift
+//  stripe-react-native
+//
+//  Created by Charles Cruzan on 6/9/22.
+//
+
+import Foundation
+import Stripe
+
+class PushProvisioningUtils {
+    class func canAddCardToWallet(
+        last4: String,
+        primaryAccountIdentifier: String,
+        testEnv: Bool
+    ) -> (canAddCard: Bool, details: NSDictionary) {
```

Comment:
Seems like we could just pass a status enum back instead of a details dictionary, wdyt?  

I'm not sure but it looks like the caller of this method is redundantly putting the details returned here into another dictionary too.